src/App.tsx

```diff
@@ -1,19 +1,126 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import './App.css';
-
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './Components/TodoList';
+import { Todo } from './Types/Todo';
+import { User } from './Types/User';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+type State = {
+  tasks: Todo[],
+  taskTitle: string,
+  userName: string,
 };
 
+const completedList: Todo[] = todos.map((todo) => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
+
+class App extends React.Component<{}, State> {
+  state = {
+    tasks: completedList,
+    taskTitle: '',
+    userName: 'Please choose a user',
+  };
+
+  createTask() {
+    if (!this.validate()) {
+      this.clearForm();
+    }
+
+    const { tasks, taskTitle, userName } = this.state;
+    let tempUser: User | null = users.find(user => user.name === userName) || null;
+
+    if (tempUser === null) {
+      tempUser = users[0];
+    }
+
+    const newTask: Todo = {
+      userId: tempUser.id,
+      id: tasks.length + 1,
```

Comment:
What if id's are not like `1,2,3` but instead are `56,142,532`? Use the maximum id and add `1` to it